Source/missiles.h

```diff
@@ -215,22 +215,22 @@ void AddStealPotions(Missile &missile, const AddMissileParameter &parameter);
 void AddManaTrap(Missile &missile, const AddMissileParameter &parameter);
 void AddSpecArrow(Missile &missile, const AddMissileParameter &parameter);
 void AddWarp(Missile &missile, const AddMissileParameter &parameter);
-void AddLightningWall(Missile &missile, const AddMissileParameter &parameter);
+void AddLightningwall(Missile &missile, const AddMissileParameter &parameter);

```

Comment:
why

Reply:
Followed the format that was more consistently used where only the first letter used in the spell name was capitalized regardless of how many words are in the spell name. I can capitalize first letter of all words if it looks better